tests/thread/README.md

```diff
@@ -16,4 +16,8 @@ Test various OE synchronization primitives:
   **oe_rwlock_t**
   1. *TestReadersWriterLock* : Tests readers-writer lock invariants by launching multiple reader and writer threads racing against each other. Asserts that multiple/all readers can be simultaneously active, only one writer is active,  readers and writers are never simultaneously active.
 
+  **oe_spinlock_t**
+  1. *TestErrnoMultiThreadsSameenclave* : Tests spinlock protecting errno in multi-threads for same enclaves.
+  1. *TestErrnoMultiThreadsDiffenclave* : Tests spinlock protecting errno in multi-threads for different enclaves.
```

Comment:
For TestErrnoMultiThreadsDiffenclave g_flag_withlock is set to 0. So, there is no spinlock protecting errno.